PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,43 @@
+#Azure SDK for Java Pull Request Template Guide
+
+Thank you for your interest in contributing to Azure SDK for Java.
+
+## Description
+Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. 
+List any dependencies that are required for this change.
+
+
+Fixes # (issue)
+
+## Type of change
+
+Please delete options that are not relevant.
+
+- [ ] Bug fix (non-breaking change with fixes an issue)
+- [ ] New feature (non-breaking change with adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected) 
+- [ ] This change requires a documentation update
+
```

Comment:
It is weird to have 'new feature (non-breaking)' and 'breaking'. I would have expected to have, e.g. 'Non-breaking bug fix', 'Breaking bug fix', 'non-breaking feature', and 'breaking feature'.